improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -55,13 +55,13 @@
     EnsembleCalibrationAssertions,
     SetupCubes,
     _create_historic_forecasts,
+    build_coefficients_cubelist,
 )
 
-try:
-    imp.find_module("statsmodels")
+STATSMODELS_FOUND = False
+statsmodels_spec = importlib.util.find_spec("statsmodels")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Constant name "statsmodels_spec" doesn't conform to UPPER_CASE naming style](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5439763)